tests/Doctrine/Tests/TestUtil.php

```diff
@@ -74,6 +78,34 @@ public static function getPrivilegedConnection(): DbalExtensions\Connection
         return $connection;
     }
 
+    public static function configureProxies(Configuration $configuration): void
+    {
+        $configuration->setProxyDir(__DIR__ . '/Proxies');
+        $configuration->setProxyNamespace('Doctrine\Tests\Proxies');
+
+        $proxyImplementation = $_ENV['ORM_PROXY_IMPLEMENTATION']
+            ?? $_SERVER['ORM_PROXY_IMPLEMENTATION']
+            ?? null;
+
+        if (! $proxyImplementation) {
+            $proxyImplementation = trait_exists(LazyGhostTrait::class) ? 'lazy-ghost' : 'common';
```

Comment:
Because I also want the fallback to kick in if the environment variable has been set to an empty string.

Reply:
Why would we do that? :thinking:

Reply:
Because unsetting environment variables is hard. If you're dealing with enviroment variables, you should always treat the empty string as if the variable wasn't set.

Fun fact: I did not take the empty string into account at first which made the CI fail: https://github.com/doctrine/orm/actions/runs/3664887468/jobs/6195591631

Reply:
I'm sorry, where are we unsetting that environment variable? And is that not as simple as `putenv('MYVAR')`?

Reply:
> I'm sorry, where are we unsetting that environment variable?

We aren't, but e.g. Github Actions needs to do that.